### DISCUSSION TITLE
comment on the xDrip screenshot

### DISCUSSION THREAD

**Discussion by Roozeboom007** [NightscoutFoundation/xDrip#2260] [Q&A]
Anyone have any idea what the comment below means, on the screenshot?

![Screenshot_20220804-094836_2](https://user-images.githubusercontent.com/58114624/182925675-5042bbf9-7fe6-4dc7-b18f-c9b064be31a8.jpg)

**Comment by jamorham** [comment]
I don't know if this pop-up message is coming from xDrip, but if it is, it would be from the Emergency Assist feature which is the only place where `getLattitude()` is called. Unless this has been enabled then I don't see there is any way xDrip would be doing anything with the gps.
